signup.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html><head>
+  <style>
+  button {
+      background-color: #4CAF50;
+      color: white;
+      margin: 8px 0;
+      border: none;
+      cursor: pointer;
+      padding: 7px 40px;
+      width: 15%;
+  }
+  
+  button:hover {
+      opacity: 0.8;
+  }
+  body {
+    background-color: #BDB76B;
+}</style>
+    
+
+<title>Maintainance Tracker</title>
+</head>
+<body>
+    <form  >
+  
+    <h1>Sign Up</h1>
+    <p>Please fill in this Your details to create an account.</p>
+
+    <table style = "width:50%">
```

Comment:
Using a <table /> isn't a good Idea.
It would be better if you use `div` and style them to look better for example

```html
<div>
  <label for="firstName">First Name</label>
  <input type="text" id="firstName" name="firstName" />
</div>
```
If you use `<label />`, Its lways a good practice to provide a `for` attribute on the label. Its value should be the Id of the input element to which its acting as a label to as done in the example above